initial PR for AI capabilities (#64)
signalwire/docs: docs/home/products/ai/get-started/capabilities/index.mdx

import AIVRFlow from './_mermaid/ai-ivr-flow.mdx';
import TraditionalSupportFlow from './_mermaid/traditional-support-flow.mdx';
import AISupportFlow from './_mermaid/ai-support-flow.mdx';

## Introduction

SignalWire's AI platform is a unified system for building and deploying conversational AI solutions. The platform combines:

- A single platform to orchestrate voice, video, and messaging channels
- Native integrations with leading LLM, Text-to-Speech, and Speech-to-Text providers
- Serverless functions that execute with minimal latency during live conversations
- Multi-threaded architecture for parallel, asynchronous function execution
- Global edge network with points of presence in every major region
- Enterprise-grade security, compliance, logging and analytics

You can use SignalWire AI as a complete agent development platform or as an orchestration layer for your existing AI solutions.

---

## Core capabilities

### Voice technology

SignalWire's voice technology gives you control over how your AI agents sound and understand speech:

- Choose from multiple Text-to-Speech providers and fine-tune voice parameters to match your brand

> - Add natural speech <Tooltips tip="A user can define what words or phrases to say during pauses in the conversation. These pauses can occur from function calls or speach down time.">fillers</Tooltips> to maintain conversation flow during processing pauses

Reviewer: correct typo
```suggestion
- Add natural speech <Tooltips tip="A user can define what words or phrases to say during pauses in the conversation. These pauses can occur from function calls or speech down time.">fillers</Tooltips> to maintain conversation flow during processing pauses
```

- Process incoming audio in real-time with noise filtering and accent handling

### Conversation intelligence

SignalWire AI helps you build agents that can handle complex conversations naturally. Unlike traditional IVR systems that follow rigid decision trees, your agents can:

- Stay on task and maintain their assigned role
- Keep track of multiple conversation threads
- Interface with multiple backend systems while maintaining natural dialogue
- Handle unexpected topic changes without losing context

> Let's compare how a customer might schedule a medical appointment:

Reviewer: ```suggestion
The below diagrams illustrate how a customer might schedule a medical appointment using a conventional IVR ("interactive voice response", left), compared to using a SignalWire AI Agent (right).
```

            Note over AI,CalAPI: Checking team availability
            AI->>+Functions: check_team_availability()
            Functions->>+CalAPI: Query sales team calendar
            CalAPI-->>-Functions: Return available slots
            Functions-->>-AI: Provide available times
            
            AI->>-Customer: The sales team is free at 2 PM or 4 PM on Tuesday
            Customer->>+AI: 2 PM works perfectly
            
            %% Meeting Setup
            Note over AI,CalAPI: Scheduling the meeting
            AI->>+Functions: schedule_meeting()
            Functions->>+CalAPI: Create event and send invites
            CalAPI-->>-Functions: Confirm booking and invites sent
            
            %% Notifications
            Note over Functions,SMS: Sending notifications
            Functions->>Functions: Trigger SMS notification
            Functions->>+SMS: Send confirmation text
            SMS-->>-Functions: Confirm delivery status
            
            Functions-->>-AI: Confirm meeting setup
            AI->>-Customer: Meeting scheduled for Tuesday at 2 PM
            Note over Customer,AI: You'll receive a calendar invite and a confirmation SMS

> ```

Reviewer: Consider putting this in a `<frame>`

The process works like this:

1. Your agent recognizes when a request needs external data or actions
2. It calls the appropriate function
3. The function handles the technical work with your systems
4. Your agent incorporates the results naturally into the conversation

This lets you automate complex processes without exposing the technical details to your users.

### SignalWire's RAG stack integration - Datasphere

> [Datasphere](/rest/signalwire-rest/endpoints/datasphere/documents) is SignalWire's built-in system for giving your AI agents access to your organization's knowledge. It helps your agents:
> 
> - Find and use relevant information during conversations
> - Keep responses accurate and up-to-date
> - Back answers with your official documentation

Reviewer: Recommend a paragraph instead of bullets here
```suggestion
[Datasphere](/rest/signalwire-rest/endpoints/datasphere/documents) is SignalWire's built-in system for giving your AI agents access to your organization's knowledge. 
Datasphere empowers your agents to find and use relevant and approved information during conversations, keeping responses accurate and up-to-date.
```
